easybuild/easyconfigs/d/deepTools/deepTools-3.1.2-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,61 @@
+
+easyblock = "PythonPackage"
+
+name = 'deepTools'
+version = '3.1.2'
+
+homepage = 'https://pypi.python.org/pypi/deepTools/'
+description = """Tools to process and analyze deep sequencing data."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['fc8b17f0a791b5ba6a3d6bcb32ec0731614da2397d38218217ec3d3cd6ee6129']
+
+python = 'Python'
+pythonversion = '2.7.14'
+pyshortver = ".".join(pythonversion.split(".")[:-1])
```

Comment:
@ebirn Please use `%(pyver)s` in `versionsuffix` (right below `version =`) and move the Python version into `dependencies` like we do in other easyconfigs, this is the old style...